src/EnergyPlus/HeatPumpWaterToWaterSimple.cc

```diff
@@ -780,98 +691,116 @@ namespace HeatPumpWaterToWaterSimple {
-                                       RoutineName);
-                GSHP(GSHPNum).SourceSideDesignMassFlow = GSHP(GSHPNum).RatedSourceVolFlowHeat * rho;
-            } else if (GSHP(GSHPNum).WWHPPlantTypeOfNum == TypeOf_HPWaterEFCooling) {
-                rho = GetDensityGlycol(PlantLoop(GSHP(GSHPNum).LoadLoopNum).FluidName,
-                                       DataGlobals::CWInitConvTemp,
-                                       PlantLoop(GSHP(GSHPNum).LoadLoopNum).FluidIndex,
-                                       RoutineName);
-                GSHP(GSHPNum).LoadSideDesignMassFlow = GSHP(GSHPNum).RatedLoadVolFlowCool * rho;
-                rho = GetDensityGlycol(PlantLoop(GSHP(GSHPNum).SourceLoopNum).FluidName,
-                                       DataGlobals::CWInitConvTemp,
-                                       PlantLoop(GSHP(GSHPNum).SourceLoopNum).FluidIndex,
-                                       RoutineName);
-                GSHP(GSHPNum).SourceSideDesignMassFlow = GSHP(GSHPNum).RatedSourceVolFlowCool * rho;
+            this->reportPower = 0.0;
+            this->reportEnergy = 0.0;
+            this->reportQLoad = 0.0;
+            this->reportQLoadEnergy = 0.0;
+            this->reportQSource = 0.0;
+            this->reportQSourceEnergy = 0.0;
+            this->reportLoadSideMassFlowRate = 0.0;
+            this->reportLoadSideInletTemp = 0.0;
+            this->reportLoadSideOutletTemp = 0.0;
+            this->reportSourceSideMassFlowRate = 0.0;
+            this->reportSourceSideInletTemp = 0.0;
+            this->reportSourceSideOutletTemp = 0.0;
+            this->IsOn = false;
+            this->MustRun = true;
+
+            if (this->WWHPPlantTypeOfNum == TypeOf_HPWaterEFHeating) {
+                rho = GetDensityGlycol(
+                    PlantLoop(this->LoadLoopNum).FluidName, DataGlobals::CWInitConvTemp, PlantLoop(this->LoadLoopNum).FluidIndex, RoutineName);
+                this->LoadSideDesignMassFlow = this->RatedLoadVolFlowHeat * rho;
+                rho = GetDensityGlycol(
+                    PlantLoop(this->SourceLoopNum).FluidName, DataGlobals::CWInitConvTemp, PlantLoop(this->SourceLoopNum).FluidIndex, RoutineName);
+                this->SourceSideDesignMassFlow = this->RatedSourceVolFlowHeat * rho;
+            } else if (this->WWHPPlantTypeOfNum == TypeOf_HPWaterEFCooling) {
+                rho = GetDensityGlycol(
+                    PlantLoop(this->LoadLoopNum).FluidName, DataGlobals::CWInitConvTemp, PlantLoop(this->LoadLoopNum).FluidIndex, RoutineName);
+                this->LoadSideDesignMassFlow = this->RatedLoadVolFlowCool * rho;
+                rho = GetDensityGlycol(
+                    PlantLoop(this->SourceLoopNum).FluidName, DataGlobals::CWInitConvTemp, PlantLoop(this->SourceLoopNum).FluidIndex, RoutineName);
+                this->SourceSideDesignMassFlow = this->RatedSourceVolFlowCool * rho;
```

Comment:
Both use CWInitConvTemp?

Reply:
No chance.  But for that initial copy/paste, sure why not?  

I appreciate you keeping your eyes on this as I'm developing it @rraustad !

Reply:
@rraustad actually, since this is the original equation fit WWHP and I am just refactoring it for future integration, I'm going to leave it as CWInitConvTemp since that's what it had before.  I'm trying to keep this PR as diff-free as possible.